### PR TITLE
Added documentation for api version 1.1

### PR DESCRIPTION
**Pull request by erikhagglund**
* Documentation for motion sensors
* Documentation for free functions
* Documentation for setting latency

Including minor styling issues with inline code

### CODE REVIEW

File: content/android/api/processing.md [hunk 1]
```diff
@@ -133,6 +188,34 @@ Corresponds to the GraphicBuffer class in Android.
...
+#### Parameters
+*backend* The GraphicBuffer
+
+*native_buffer* The native buffer
+
+*handle* The buffer handle
+
+*size* Image resolution.
+
+*pixel_stride* Image stride in bytes
```
Comment (emilwestergren) [inline content/android/api/processing.md]: This is pixel stride, i.e. not in bytes unless `1 pixel = 1 byte`

Comment (emilwestergren) [inline content/android/api/processing.md]: This is pixel stride, i.e. not in bytes unless `1 pixel = 1 byte`

Comment (emilwestergren) [inline content/android/api/processing.md]: So "Image stride in pixels"

File: content/android/api/processing.md [hunk 2]
```diff
@@ -133,6 +188,34 @@ Corresponds to the GraphicBuffer class in Android.
...
+
+*format* The buffer format
+
+---
+### vidhance_graphic_buffer_t
+Frees a vidhance_graphic_buffer_t, normaly not needed since ownership is transfered to vidhance when creating an image
```
Comment (emilwestergren) [inline content/android/api/processing.md]: *transferred*

File: content/android/api/settings.md [hunk 3]
```diff
@@ -4,25 +4,45 @@
...
 Creates a new Vidhance settings object.
-```
+``` c
 vidhance_settings_t vidhance_settings_new();
 ```
 ---
+### vidhance_settings_set_latency
+Set latency
+``` c
+void vidhance_settings_set_latency(const vidhance_settings_t settings, int frames);
```
Comment (emilwestergren) [inline content/android/api/settings.md]: Not released yet

**Comment by erikhagglund** [comment]
Updated according to Emils comments

File: content/android/api/processing.md [hunk 4]
```diff
@@ -133,6 +188,34 @@ Corresponds to the GraphicBuffer class in Android.
...
+
+*size* Image resolution.
+
+*pixel_stride* Image stride in pixels
+
+*format* The buffer format
+
+---
+### vidhance_graphic_buffer_t
+Frees a vidhance_graphic_buffer_t, normaly not needed since ownership is transferred to vidhance when creating an image
```
Comment (emilwestergren) [inline content/android/api/processing.md]: **normally**

**Comment by emilwestergren** [comment]
Will merge on release 1.1

**Comment by erikhagglund** [comment]
updated spelling error

**Pull request merged**